src/frontend/src/components/public/PIAListTable/TableOrdering.tsx

```diff
@@ -1,15 +1,28 @@
+import { useEffect, useState } from 'react';
 import SortAscending from '../../../assets/Sort-ascending.png';
 import SortDescending from '../../../assets/Sort-descending.png';
 import SortInactive from '../../../assets/Sort-inactive.png';
 import { PiaSorting } from '../../../constant/constant';
 
 interface ITableOrderingProps {
   currentState: number;
+  title: string;
 }
+type SortOrder = 'none' | 'ascending' | 'descending' | 'other' | undefined;
 
 const TableOrdering = (props: ITableOrderingProps) => {
-  const { currentState } = props;
+  const { currentState, title } = props;
+  const [sortingLabel, setSortingLabel] = useState<SortOrder>('none');
 
+  useEffect(() => {
+    setSortingLabel(
+      currentState === PiaSorting.INACTIVE
+        ? 'none'
+        : currentState === PiaSorting.DESCENDING
+        ? 'descending'
```

Comment:
should this be more descriptive? Last modified ascending/descending or drafter ascending/descending?

Reply:
add to aria-label